transform-plugins/docs/ValueMapper-transform.md

```diff
@@ -0,0 +1,99 @@
+# ValueMapper Transform
+
+
+Description
+-----------
+Value Mapper is a transform plugin that maps string values of a field in the input record 
+to mapping value using mapping dataset.
+
+Mappings for the values are usually stored in a key-value dataset. The ValueMapper transform
+provides a simple method for manipulating input data, both a field and its values, using a mapping.
+
+
+Use Case
+--------
+One use is to replace language codes in the input record field with an 
+appropriate language description:
+     **Source field name:** language_code
+     **Target field name:** language_desc
+     **Mappings, source to target:** DE/German, ES/Spanish, EN/English
+
+This will replace the source column *language_code* with the target column 
+*language_desc*, replacing values found in the source field using the mappings 
+("DE" to "German", "ES" to "Spanish", and so on.)
+
+
+Properties
+----------
+**mapping:** A comma-separated list that defines the mapping of a source
+field to a target field and the mapping table name for looking up values. 
+Contains three fields separated by a colon (":") as the source field, the 
+mapping table name, and the target field:
+
+         <source-field>:<mapping-table-name>:<target-field>
+
+Note: **source field** supports only STRING types.
```

Comment:
Given that source field only supports STRING types, does that mean that the keys all have to be strings, though the mapped values could be a different type, if the column it was mapped to (the target field) supported it?